external/VMS_DB_Backup/scripts/VmsDbBackupCon.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+HOME="/backups"
+now="$(date +'%d_%m_%Y_%H')"
+fileName="db_backup_$now".gz
+backupFolder="$HOME/database_backup"
+fullPathBackupFile="$backupFolder/$fileName"
+logfile="$backupFolder/backup_log_$(date +'%Y_%m').txt"
+vmsBackupTag="[VMS Backup Process]"
+
+mysqlPassword=$(cat $HOME/scripts/config/mysqlPassword.conf)
```

Comment:
One last comment. Shall we move all configs to one file:
```
mysql_username=example
mysql_password=example
```

and then use something like following to read individual value:
```
$(cat $HOME/scripts/config.conf | grep "mysql_username" | cut -d '=' -f2)
```